yubihsm-container/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# SPDX-FileCopyrightText: 2023 Technology Innovation Institute (TII)
+# SPDX-License-Identifier: Apache-2.0
+
+FROM ubuntu:jammy
```

Comment:
For future it might be desirable to have a light-weight container, then an alpine based image might be an option.
Of course that would require testing, I'm not sure if the yubitools would work there.
I suppose there would be also a slimmer ubuntu image that you could use, that would work more likely.

Reply:
Yes, let's research this. (combining with https://github.com/tiiuae/ci-yubi/pull/1#discussion_r1599438878 )